test/acts_as_tree_test.rb

```diff
@@ -261,7 +267,7 @@ def test_is_leaf
   end
 
   def test_tree_view
-    assert_equal false, Mixin.respond_to?(:tree_view)
+    assert_equal false, TreeMixin.respond_to?(:tree_view)
     Mixin.extend ActsAsTree::TreeView
     assert_equal true,  TreeMixin.respond_to?(:tree_view)
```

Comment:
This tests that `Mixin` is properly extended with `ActsAsTree::TreeView`, so `Mixin` should be used in both cases (meaning both the existing and your changes version seem to be wrong).